seqr/utils/es_utils.py

```diff
@@ -564,6 +564,7 @@ def _parse_compound_het_response(self, response):
                 for family_guid in variant['familyGuids']:
                     family_variants[family_guid].append(variant)
 
+            # TODO place compound hets from the same family together
```

Comment:
this seems important?

Reply:
`.append(variant)` seems to implicitly ensure that compound hets from the same family are placed together.

Reply:
Need to design a new UI to label compound hets from different families.

Reply:
^ Display in pairs